BIOPHOTOVOLTICS (Home)
 
Solar energy systems have emerged as the predominant player in renewable energy technologies. However, solar is not a perfect technology. 95% of current solar panels use silicon as a semiconductor [1]. However, mining transporting, and refining the raw material is extremely energy and labour-intensive [2]. Additionally, large-scale solar farm installations often necessitate the destruction of fields, incurring a heavy ecological burden on their surroundings. Further to this are issues pertaining to recycling and the life cycle of current panels being 20-25 years and non-circular [3]. There are few current or planned efforts to deal with the significant volumes of silicon waste from degraded panels, and this issue will pose significant ecological burdens in the near future [3].
The field of bio-photovoltaics (BPVs) represents a groundbreaking approach that harnesses the capabilities of photosynthetic microorganisms, such as cyanobacteria or algae, to generate electricity from CO2 and sunlight. 

>>> Delete Figure 1. Graph showing the annual additions of Renewable power capacity.

How does it work?

 
Figure 1. Schematic Representation of Bio-Photovoltaic Electricity Generation. The bio-photovoltaic panel utilizes photosynthesis to produce electricity while actively sequestering carbon. Within the anodic chamber, light triggers photosynthetic activity in the algae, leading to the flow of excited electrons through the oxidative electrode (anode). This sets in motion a process where protons can penetrate a selective membrane. Concurrently, these excited electrons journey towards the reductive electrode (cathode) in the cathodic chamber, propelling an electric current through the external circuit.[4]
 

Figure 2. Putative routes for electron transfer from thylakoid membrane constituents (photosystems) to an anode. The routes described in the diagram are both mediated electron transfer and potential transfer mechanisms through pili or other membrane configurations. The dashed red routes are conjectured electron transfer pathways. Top right: a schematic depiction of a prototypical BPV arrangement. [5]

Figure 2. Demonstrates a model for a potential biophotovoltaic cell. Despite their potential, bio-photovoltaics face challenges with low conversion efficiencies primarily attributed to inefficient extracellular electron transfer (EET) across the outer cell membrane [6]. Furthermore, the precise mechanisms governing electron transfer between microorganisms and the electrode remain poorly understood [7]. Figure 2 shows some of the pathways that electrons could be transferred from the thylakoid membrane to the anode where the electrons can be harvested for electricity.

Benefits of BPVs

1. Cyanobacteria culture releases less CO2 in comparison to silicon mining.
2. Cyanobacteria are biodegradable. In contrast, it is difficult to recycle solar panels due to their complex composition and toxic materials.
3. BPVs can achieve carbon neutrality as CO2 is consumed to produce electricity.

Our Solution

Project CynergisED aims to address the shortcomings of solar panels by designing an optimized BPV. The different parts of our project each take on issues facing conventional solar panels and BPVs, aiming to produce a product that contributes to a circular economy.

Conductivity
One of the biggest barriers to the success of BPVs is power output. The highest reported power density to date was 610 mW/m2 using Synechococcus sp. BDU 140432, achieving a 4% light-to-power efficiency in contrast to the 23% achieved by the highest efficiency solar panel [8]. Therefore, our MTR and salinity projects focused on increasing the power output of our BPV.

Recycling/Conservation 
With an understanding that disruptions to food chains and ecosystems are inevitable consequences of large solar farms, we are engineering Synechocystis sp. PCC 6803 for a pollinator-friendly BPV—a subproject intricately woven into the fabric of a circular economy. 

Biosafety
It is important to consider how our GMO-driven solar panel could fit in the real world and what potential challenges in relation to biosecurity and GMO regulations it could face. Given that a BPV installation would be placed outdoors, there is potential for the GM cells to escape into the environment. Thus, to receive the government approval of the product, would be essential to implement additional containment strategies.
Conductivity - The MTR Complex  

Previous studies have consistently demonstrated low photocurrent production in wild-type cyanobacteria and algae [6], [9], [10], [11]. The challenge in enhancing this process lies in the incomplete understanding of extracellular electron transfer (EET) mechanisms. In contrast, the non-photosynthetic metal-reducing microorganism Shewanella oneidensis MR-1 boast a well-characterised EET pathway involving the MtrCAB complex [12], [13], making it ideal for bioelectricity applications [14]. 

>> delete the crystal structure of mtr

The heightened photocurrent observed in Synechocystis sp. PCC 6803 cells expressing MtrA was directly attributed to MtrA expression, prompting the hypothesis that further enhancements in photocurrent could be achieved by expressing outer membrane proteins, such as MtrB, MtrC, OmcS, and CymA. This expression would facilitate direct electron transfer to the electrode surface and is the focus of this subproject.

The electron transfer pathway (Fig. 3) remains an area requiring comprehensive understanding. Current knowledge suggests that electrons, derived from the substrate's oxidation, ultimately find their way to the periplasmic decaheme cytochrome MtrA during anaerobic respiration [15]. In the subsequent step, electron transfer to the outer membrane β-barrel protein MtrB establishes direct contact between MtrA and MtrC. MtrC, in turn, undergoes re-oxidation through the reduction of terminal electron acceptors, such as electrodes, achieved via either direct or mediated electron transfer. The former demands physical contact between the electrode and outer membrane redox proteins, while the latter involves a soluble electron shuttle capable of being reduced by outer membrane cytochromes and ferrying the electrons to the electrode in a reduced state.

A. 							B.
  
Figure 3. Cym/MtrCAB complex. This complex is composed of MtrA and MtrC, two decaheme cytochromes, inside a transmembrane porin, MtrB, which transports electrons across the outer membrane to electron acceptors, in this case, an electrode. 3A. Electron transport through the complex. Include CymA significantly improves cell viability in E. coli. Image from Dundas et al. paper[15] 3B. Heme groups in Mtr complex. MtrA is shown in green, and MtrC is shown in blue. The β sheets of the MtrB subunit are shown in black, with the solvent-exposed loops in grey; red symbols are Fe in heme groups in MtrA and MtrC. Image from White et al [16].

Conductivity – Salt Tolerance

Another way of improving the flow of electrons through a BPV is by adding electrolytes like NaCl to the growth medium. 

From our human practices with Dr Jamie Marland, we learned that an effective way of improving the power output of a BPV is to add more electrolytes to the medium which can carry charge and improve current flow. The ideal electrolyte for a potential cell is cheap, will not interfere with the medium and is non-toxic to the cyanobacteria cells. We decided that the best electrolyte in this case was NaCl. 
Jamie Marland explained to us that as more electrolytes like NaCl are added to the medium the conductivity will increase. Synechocystis sp. PCC 6803 is the model organism for cyanobacteria, with a fully sequenced and well-annotated genome. Much of the research into biophotovoltaics utilises synechocystis due to these advantages. However, synechocystis sp. PCC 6803 is a cyanobacteria strain which is considered to have only intermediate tolerance to salt amongst cyanobacteria. Cyanobacteria strains such as those isolated from marine environments. However, these strains are not as well annotated and understood as synechocystis. Therefore, we decided to attempt to generate a more salt-tolerant strain of synechocystis to be able to add as much salt to our medium as possible, without having a negative impact on our synechocystis.
Circular Economy - Artificial Pollen 

CynergisED goes beyond merely producing clean energy; it involves devising a solution for the recycling of waste generated during the energy production stage as well as mitigation strategies for the biodiversity loss associated with solar farms. With this in mind, we focused on repurposing BPV cell culture waste into artificial pollen. Beekeepers commonly utilise commercially available pollen replacements – such as cheap sucrose solutions and Brewer’s yeast [18]. However, these supplements have a lower nutritional content than natural pollen (Figure 4). The cyanobacteria genus Spirulina has emerged as a potential substitute due to its photosynthetic activity [19]. In comparison to pollen, Spirulina contain similar content of all amino acids except for histidine and lysine (Figure. 4).  We therefore hypothesized that increasing the expression of these two amino acids in cyanobacteria would make it a more nutritious pollen substitute.

  
Figure 4. The concentrations of essential amino acids in various diets for pollinators, including pollen. Ultra Bee refers to a commercial plant-based pollen substitute. Image from Ricigliano et al [17].

Histidine Biosynthesis
The biosynthesis pathway of histidine (BPH) in bacteria is a complex and highly regulated process that involves several enzymatic reactions, being a model system for studying relationships between enzymes and intermediate flow in bacteria (Figure 5). The BPH typically starts with the conversion of 5-phosphoribosyl-1pyrophosphate (PRPP) and ATP to phosphoribosyl-ATP (PR-ATP) through the enzyme phosphoribosyl pyrophosphate synthetase (PRPP synthetase), regulated by the protein HisG. Subsequent steps involve several enzymatic reactions, including the transformation of PR-ATP to imidazole ribotide, followed by the formation of histidine monophosphate (HMP) and histidinol. The final conversion of histidinol to histidine is catalysed by histidinol dehydrogenase. This pathway is tightly regulated by feedback inhibition, ensuring that the production of histidine is balanced with the cellular requirements.
 
 
Figure 5: Schematic representation of proposed enhancements in the biosynthesis pathway of L-histidine in Cyanobacteria for this project. The diagram represents the first stage of the L-histidine pathway in C. glutamicum, adapted from [20], with annotations by author. 


Lysine Biosynthesis 

In bacteria, dapA is a key gene involved in the first stages of lysine biosynthesis.  It encodes the enzyme dihydrodipicolinate synthase (DapA) and individual expression of dapA has been shown to increase the production of lysine within Bacillus methanolicus and for this reason, we chose to overexpress DapA (Figure 6).

 

Figure 6: Proposed increased activities of relevant genes within the BPL of Cyanobacteria for this project. Stages show lysine biosynthesis in E. coli, adapted from [21], annotations by the author.

Biosafety

For the genetic containment of our system, we designed and built a kill switch to prevent the escape and survival of GMO Synechocystis in the environment. The kill switch is based on the constitutive expression of a toxin, NucA nuclease [22], from a different cyanobacterium Anabaena sp. PCC 7120 together with its inhibitor NuiA [23], which we put under the control of a zinc-inducible promoter. We chose to use zinc as the inducer because Synechocystis tolerance to Zn2+ ions is relatively high in comparison to other heavy metals with IC50 (half-growth inhibitory concentration) ranging between 8 and 16 µM Zn2+ [24]. Moreover, it was also found that Synechocystis had a native promoter regulating the copMRS operon involved in copper response and that this promoter could also be induced by zinc [25].
 
NucA/NuiA Kill Switch in Synechocystis.
NucA is a non-specific DNA/RNA nuclease from the cyanobacterium Anabaena sp. PCC 7120 that can cut single-stranded and double-stranded DNA and RNA [22]. Nucleases of this type are present in several bacterial species and are believed to have evolved to serve nutritional purposes and sometimes as bacteriocides [23]. NucA requires divalent metal ions like Mn2+ or Mg2+ as cofactors, the optimal concentration for these being around 5 mM [23]. NucA activity was shown to decrease with increasing concentration of monovalent salt [23]. NucA contains a ββα metal finger motif and a hydrated divalent metal ion at the active site [22]. Ghosh et al. (2005) proposed that His124 acts as a catalytic base, and Arg93 participates in the catalysis possibly through stabilization of the transition state. NucA forms a 1:1 complex with its specific inhibitor NuiA (Figure 2) from the same bacterium Anabaena sp. PCC 7120, leading to complete inhibition of NucA [23]. The inhibition involves an unusual divalent metal ion bridge between the nuclease and its inhibitor [26]. The C-terminal Thr-135 hydroxyl oxygen in NuiA interacts directly with the catalytic Mg2+ in the nuclease active site, while Glu-24 in NuiA extends into the active site, mimicking the charge of a scissile phosphate [26]. NuiA residues Asp-75 and Trp-76 contribute to the strength and specificity of the interaction [26].
 
NucA-NuiA complex. Taken from Ghosh et al. (2007).
Our idea is that within the solar panel, the growth medium would be supplemented with Zn2+ ions to induce NuiA antitoxin expression, thus counteracting NucA nuclease effects. In the environment, Zn2+ concentration would usually be too low for efficient NuiA induction, thus NucA would cut cellular DNA and RNA, killing escaped cells. One of the reasons why we chose this specific nuclease as a toxin in our system is because it is non-specific, meaning that it can cut all nucleic acids, including single-stranded and double-stranded, DNA and RNA [22]. Thus, it has the potential not only to kill the escaped GM bacteria, but also prevent the horizontal gene transfer. Importantly, we designed and built the kill switch on a plasmid due to the time constraints of the iGEM competition, however, bacteria can easily lose plasmids, thus ideally in a real-world scenario, the kill switch would have to be incorporated into the genome. Moreover, it would be necessary to remove all antibiotic resistance genes that are generally used for selection in cloning experiments to avoid the potential spread of these genes to other organisms in the environment.

Despite the neat design of many genetic containment systems, including kill switches, there are multiple problems associated with them. First, random mutations in any component of the kill switch can inactivate it, for example, in our case, mutations in the zinc promoter could impair its inducibility and lead to constitutive expression of antitoxin allowing cell survival in the environment, or mutations in the nuclease could impair its structure inactivating it or reducing its toxicity. In addition, when testing for the elimination of bacteria, it should be noted that available methods for monitoring cell survival, like optical density measurements of bacterial growth and colony-forming unit counting for measuring cell viability, are typically done in carefully controlled experiments, which may fail to capture mechanisms of escape that may arise in the more complex natural environment, and thus cannot prove complete eradication of genetically engineered strains, as there is always a possibility that the cells could be surviving in a dormant state.

Cyanogate
CyanoGate, developed by the McCormick lab at the University of Edinburgh, facilitates the generation of knock-outs, integrations, and multigene expression systems in cyanobacteria [27].
Cyanobacteria are crucial in their ecological importance and potential biotechnological applications, and have been underutilized in synthetic biology. CyanoGate aims to bridge this gap by providing a suite of genetic parts and acceptor vectors, enabling integrative or replicative transformation. Tested in model cyanobacteria, Synechocystis sp. PCC 6803 and Synechococcus elongatus UTEX 2973, CyanoGate demonstrates its applicability for efficient genetic modification.

The importance of cyanobacteria in biotechnology, from pharmaceuticals to biophotovoltaic devices, underscores the need for robust synthetic biology tools. Despite progress, cyanobacteria lag behind in synthetic biology compared to other systems. CyanoGate addresses this challenge by offering an easy-to-use system that integrates with existing standards, enhancing scalability and accelerating the "design, build, test, and learn" cycle.
The syntax of level 0 parts was adapted for prokaryotic cyanobacteria, addressing typical cloning needs in cyanobacterial research. New level 0 parts were assembled from various sources. Level 1, M and P acceptor vectors from the MoClo Plant Tool Kit facilitated the assembly of level 0 parts in a level 1 vector, with up to seven level 1 modules in level M. Level M assemblies could be further combined into level P and cycled back into level M for larger multimodule vectors if needed.
Modules from level 1 or level P could be assembled in new level T vectors designed for cyanobacterial transformation. Both UTEX 2973 and Synechocystis produced recombinants through electroporation or conjugation methods with level T vectors, with a preference for the conjugation approach in the outlined work.
 
Level 0 (Lv0) Cyanogate vector assembly from either genomic extraction or synthesised G-Block. A Lv0 Acceptor vector containing a prokaryotic selection marker (pink) and LacZ gene insert (yellow) flanked by restriction sites and the G-block of DNA are digested and ligated with BbsI. The reaction result is a transformed vector containing the DNA sequence of interest in the position of LacZ (blue).

 
Level 1 (Lv1) Cyanogate vector assembly from three Lv0 vectors containing a promotor, GOI (gene of interest) and terminator. A Lv1 Acceptor vector containing a prokaryotic selection marker (pink) and LacZ gene insert (cyan) flanked by restriction sites and the G-block of DNA is digested and ligated with BsaI. The reaction result is a transformed vector containing the DNA sequence of interest in the position of LacZ (blue).



 
Level T (LvT) Cyanogate vector assembly from two Lv1 vectors containing a Lv1 Cassette, and Lv1 End Linker. A LvT Acceptor vector containing a prokaryotic selection marker (pink) and LacZ gene insert (green) flanked by restriction sites and the G-block of DNA is digested and ligated with BbsI. The reaction result is a transformed vector containing the DNA sequence of interest in the position of LacZ (blue).

 
LvT Cassettes contain origins of replication for both E.coli and Cyanobacteria, allowing for assembly and testing in E.coli strains before transforming via triparental conjugation or electroporation into a cyanobacteria host. 


Bottom of Form

Early Projects
The following projects were discontinued due to time constraints and a desire to consolidate the project. 
Conventional photovoltaic panels can only capture light energy but currently have no scalable means to store this energy. In contrast, living biophotovoltaic systems which use photosynthetic cyanobacteria are capable of both capturing and storing energy in the form of carbohydrates and other polymers [28]. This process can further sequester or cycle carbon dioxide, helping to make our solar panel carbon negative.
Consequently, we pursued increasing the biosynthesis of polyhydroxybutyrate (PHB) a thermoplastic polyester with excellent biodegradability that is capable of being degraded by various microorganisms living in soil and saltwater. PHB acts as a carbon sink, funnelling excess metabolic flux into its production whilst sequestering CO2. PHB is native to Synechocystis sp PCC. 6803 but is only expressed in small granules in small concentrations. 
A two-pronged approach for PHB was explored: Overexpression of phaAB and directed evolution of phaB. By undergoing error-prone PCR on phaB of the phaCAB operon involved in PHB biosynthesis we hoped to generate a mutant library for the rate-limiting step to increase PHB production in E.coli initially and then transfer to  Synechocystis sp PCC. 6803.

PHB Biosynthesis
Examining the biosynthesis pathway of Polyhydroxybutyrate (PHB) in Synechocystis sheds light on the pivotal role of acetyl-CoA in a three-step reaction leading to PHB production [29]. The process initiates with 3-ketothiolase catalyzing the Claisen condensation of two acetyl-CoA molecules, yielding acetoacetyl-CoA. Subsequently, acetoacetyl-CoA transforms into R-3-hydroxybutyryl-CoA under the influence of acetoacetyl-CoA reductase. Finally, PHA synthase orchestrates the polymerization of R-3-hydroxybutyryl-CoA. Genes phaA (3-ketothiolase), phaB (acetoacetyl-CoA), phaC, and phaE (PHA synthase subunits) encode the enzymes for these critical steps.
In the pursuit of enhancing PHB production, attention has been directed towards the phaB gene. This avenue was explored by overexpressing the PHA synthase using the Ralstonia eutropha operon in Synechocystis sp. PCC6803. While the PHA synthase witnessed a two-fold increase in expression, the resulting PHB content only exhibited marginal improvement. This observation suggests that the polymerization reaction catalyzed by PHA synthase may not be the primary rate-limiting step in the PHB synthesis pathway.
Overexpressing the native pha operon in Synechocystis PCC6803 led to the highest PHB productivity. Notably, the genes phaA and phaB were located in the same operon, and focusing on the phaB gene contributed to the success of this approach. This strategy, which emphasizes the phaB gene, holds promise for further optimizing PHB production through targeted genetic modifications in the biosynthesis pathway.


Directed Evolution of PHB Pathway 
Error-prone PCR is a commonly use approach to generate mutant libraries using low-fidelity DNA polymerases such as Taq M0267 lacking 3’-5’ exonuclease proof-reading abilities Increasing MgCl­2 concentrations further decreases fidelity, increasing mutation rates. 
To generate a mutant library for phaB, the rate-limiting gene involved in PHB biosynthesis, a plasmid construct was designed with the complete phaCAB operon. This construct was derived from the iGEM part BBa_K1149051 which contained a hybrid constitutive and native promotor and the phaCAB operon. This part was optimised for bioplastic production in E.coli MG1655 and for directed evolution, the strong promotor would enable easier quantification of PHB production using Nile red fluorescence. We modified the part to contain BsaI overhangs, which would enable us to digest and ligate the ordered G-block into a level 1 plasmid backbone (DVK_AE) from the Cyanogate parts kit we were provided with thanks to the MacCormick lab. Additionally, we included a double terminator (BBa_B0015) in the G-block design. Two primer pairs, one for replicating the plasmid backbone and phaCA, and a second pair for phaB were designed. Two PCR reactions would have been run, one high-fidelity reaction to replicate a linearised backbone for phaCA, and a separate epPCR reaction with low-fidelity polymerase to generate a library of phaB mutants. The two reaction products would then be digested and ligated together to form a circularised level 1 plasmid for phaCAB* where * denotes a mutagenized gene. This would have been transformed into E.coli and expression of PHB would have been quantified by a fluorescent plate reader-based assay using Nile red. Unfortunately, we planned to synthesize the G-block as a level 1 insert from IDT but our level 1 insert was 4.43kb. IDT only synthesizes G-blocks up to 3kb in size and so we were forced to either redesign or abandon this project. Due to the added complexity of having a dual plasmid system with one containing phaCA and another containing phaB, we decided to focus efforts on salinity resistance. The main issue with dual plasmids was that one would likely be in level 1, and another in level T (containing origins of replication for cyanobacteria as well as E.coli) to prevent plasmid loss from the same origin of replication. Additionally, generating a mutant library with both plasmids by co-transformation would have added extra complexity to the project. Due to time pressures and a desire to focus on one directed evolution project, we didn’t follow through with this project.

 

Hybrid Promotor phaCAB Plasmid Assembly. Comprising of the parts BBa_K1149051 and BBa_B0015 inserted into the level 1 plasmid backbone DVK_AE from the Cyanogate kit. A kanamycin selection marker is present (KanR). BBa_J23104 is a constitutive promotor followed by a native promotor. BBa_B0034 is a ribosome binding site.

 

Figure 2: Designed level 1 G-Block for digestion and ligation into level 1 plasmid backbone DVK_AE part colours correspond to figure 1.
 
References
Bibliography
[1] Office of Energy Efficiency and Renewable Energy, “Solar Photovoltaic Cell Basics,” U.S. Department of Energy. Available: https://www.energy.gov/eere/solar/solar-photovoltaic-cell-basics#:~:text=Silicon%20is%2C%20by%20far%2C%20the
[2] S. Pizzini, “Towards solar grade silicon: Challenges and benefits for low cost photovoltaics,” Solar Energy Materials and Solar Cells, vol. 94, no. 9, pp. 1528–1533, Sep. 2010, doi: https://doi.org/10.1016/j.solmat.2010.01.016
[3] U.S. Environmental Protection Agency, “End-of-Life Solar Panels: Regulations and Management,” www.epa.gov, Aug. 23, 2021. Available: https://www.epa.gov/hw/end-life-solar-panels-regulations-and-management#:~:text=Basics%20Web%20Page%20.-
[4] R. A. Soni, K. Sudhakar, and R. S. Rana, “Biophotovoltaics and Biohydrogen through artificial photosynthesis: an overview,” International Journal of Environment and Sustainable Development, vol. 15, no. 3, p. 313, 2016, doi: https://doi.org/10.1504/ijesd.2016.077391
[5] J. Tschörtner, B. Lai, and J. O. Krömer, “Biophotovoltaics: Green Power Generation From Sunlight and Water,” Frontiers in Microbiology, vol. 10, Apr. 2019, doi: https://doi.org/10.3389/fmicb.2019.00866. Available: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6503001/
[6] Y. Zou, J. Pisciotta, R. B. Billmyre, and I. V. Baskakov, “Photosynthetic microbial fuel cells with positive light response,” Biotechnology and Bioengineering, vol. 104, no. 5, pp. 939–946, Dec. 2009, doi: https://doi.org/10.1002/bit.22466
[7] H. Schneider, B. Lai, and J. O. Krömer, “Utilizing Cyanobacteria in Biophotovoltaics: An Emerging Field in Bioelectrochemistry,” Advances in Biochemical Engineering / Biotechnology, pp. 281–302, Jan. 2022, doi: https://doi.org/10.1007/10_2022_212
[8] S. Kaushik, Mrinal Kumar Sarma, and P. Goswami, “FRET-guided surging of cyanobacterial photosystems improves and stabilizes current in photosynthetic microbial fuel cell,” Journal of materials chemistry. A, Materials for energy and sustainability, vol. 5, no. 17, pp. 7885–7895, Jan. 2017, doi: https://doi.org/10.1039/c7ta01137g
[9] N. Sekar, Y. Umasankar, and R. P. Ramasamy, “Photocurrent generation by immobilized cyanobacteria via direct electron transport in photo-bioelectrochemical cells,” Physical Chemistry Chemical Physics, vol. 16, no. 17, p. 7862, 2014, doi: https://doi.org/10.1039/c4cp00494a
[10] P. Bombelli et al., “Surface morphology and surface energy of anode materials influence power outputs in a multi-channel mediatorless bio-photovoltaic (BPV) system,” Physical Chemistry Chemical Physics, vol. 14, no. 35, pp. 12221–12229, Aug. 2012, doi: https://doi.org/10.1039/C2CP42526B. Available: https://pubs.rsc.org/en/content/articlelanding/2012/CP/c2cp42526b. [Accessed: May 17, 2022]
[11] Guillaume Longatte et al., “Investigation of photocurrents resulting from a living unicellular algae suspension with quinones over time,” Chemical Science, vol. 9, no. 43, pp. 8271–8281, Jan. 2018, doi: https://doi.org/10.1039/c8sc03058h
[12] E. Marsili, J. B. Rollefson, D. B. Baron, R. M. Hozalski, and D. R. Bond, “Microbial Biofilm Voltammetry: Direct Electrochemical Characterization of Catalytic Electrode-Attached Biofilms,” Applied and Environmental Microbiology, vol. 74, no. 23, pp. 7329–7337, Dec. 2008, doi: https://doi.org/10.1128/aem.00177-08
[13] A. A. Carmona-Martinez, F. Harnisch, L. A. Fitzgerald, J. C. Biffinger, B. R. Ringeisen, and U. Schröder, “Cyclic voltammetric analysis of the electron transfer of Shewanella oneidensis MR-1 and nanofilament and cytochrome knock-out mutants,” Bioelectrochemistry, vol. 81, no. 2, pp. 74–80, Jun. 2011, doi: https://doi.org/10.1016/j.bioelechem.2011.02.006
[14] L. Shi, T. C. Squier, J. M. Zachara, and J. K. Fredrickson, “Respiration of metal (hydr)oxides by Shewanella and Geobacter: a key role for multihaem c-type cytochromes,” Molecular Microbiology, vol. 65, no. 1, pp. 12–20, Jul. 2007, doi: https://doi.org/10.1111/j.1365-2958.2007.05783.x. Available: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1974784/#b44. [Accessed: Apr. 07, 2022]
[15] C. M. Dundas, D. Walker, and B. K. Keitz, “Tuning Extracellular Electron Transfer by Shewanella oneidensis Using Transcriptional Logic Gates,” ACS Synthetic Biology, vol. 9, no. 9, pp. 2301–2315, Jul. 2020, doi: https://doi.org/10.1021/acssynbio.9b00517
[16] G. F. White et al., “Rapid electron exchange between surface-exposed bacterial cytochromes and Fe(III) minerals,” Proceedings of the National Academy of Sciences, vol. 110, no. 16, pp. 6346–6351, Mar. 2013, doi: https://doi.org/10.1073/pnas.1220074110
[17] V. A. Ricigliano et al., “Honey Bee Proteome Responses to Plant and Cyanobacteria (blue-green algae) Diets,” ACS food science & technology, vol. 1, no. 1, pp. 17–26, Dec. 2020, doi: https://doi.org/10.1021/acsfoodscitech.0c00001
[18] S. Ahmad, Khalid Ali Khan, Shahmshad Ahmed Khan, H. A. Ghramh, and Adem Gül, “Comparative assessment of various supplementary diets on commercial honey bee (Apis mellifera) health and colony performance,” PLOS ONE, vol. 16, no. 10, pp. e0258430–e0258430, Oct. 2021, doi: https://doi.org/10.1371/journal.pone.0258430
[19] V. A. Ricigliano and M. Simone-Finstrom, “Nutritional and prebiotic efficacy of the microalga Arthrospira platensis (spirulina) in honey bees,” Apidologie, vol. 51, no. 5, pp. 898–910, May 2020, doi: https://doi.org/10.1007/s13592-020-00770-5
[20] R. K. Kulis‐Horn, M. Persicke, and J. Kalinowski, “Histidine biosynthesis, its regulation and biotechnological application in C orynebacterium glutamicum,” Microbial Biotechnology, vol. 7, no. 1, pp. 5–25, Apr. 2013, doi: https://doi.org/10.1111/1751-7915.12055
[21] Shota Isogai and H. Takagi, “Enhancement of lysine biosynthesis confers high-temperature stress tolerance to Escherichia coli cells,” Applied Microbiology and Biotechnology, vol. 105, no. 18, pp. 6899–6908, Aug. 2021, doi: https://doi.org/10.1007/s00253-021-11519-0
[22] M. Ghosh, Gregor Meiß, A. Pingoud, R. E. London, and L. C. Pedersen, “Structural Insights into the Mechanism of Nuclease A, a ββα Metal Nuclease from Anabaena,” Journal of Biological Chemistry, vol. 280, no. 30, pp. 27990–27997, Jul. 2005, doi: https://doi.org/10.1074/jbc.m501798200
[23] Gregor Meiß, I. Franke, Oleg Gimadutdinow, Claus Urbanke, and A. Pingoud, “Biochemical characterization of Anabaena sp. strain PCC 7120 non-specific nuclease NucA and its inhibitor NuiA,” European journal of biochemistry, vol. 251, no. 3, pp. 924–934, Feb. 1998, doi: https://doi.org/10.1046/j.1432-1327.1998.2510924.x
[24] B. Blasi, “Characterization of Stress Responses of Heavy Metal and Metalloid Inducible Promoters in Synechocystis PCC6803,” Journal of Microbiology and Biotechnology, vol. 22, no. 2, pp. 166–169, Feb. 2012, doi: https://doi.org/10.4014/jmb.1106.06050
[25] H. Čelešnik et al., “Biosafety of biotechnologically important microalgae: intrinsic suicide switch implementation in cyanobacterium Synechocystis sp. PCC 6803,” Biology Open, vol. 5, no. 4, pp. 519–528, Mar. 2016, doi: https://doi.org/10.1242/bio.017129. Available: https://journals.biologists.com/bio/article/5/4/519/1660/Biosafety-of-biotechnologically-important. [Accessed: Apr. 26, 2021]
[26] M. Ghosh, G. Meiss, A. Pingoud, R. E. London, and L. C. Pedersen, “The Nuclease A-Inhibitor Complex Is Characterized by a Novel Metal Ion Bridge,” Journal of Biological Chemistry, vol. 282, no. 8, pp. 5682–5690, Feb. 2007, doi: https://doi.org/10.1074/jbc.m605986200
[27] R. Vasudevan et al., “CyanoGate: A Modular Cloning Suite for Engineering Cyanobacteria Based on the Plant MoClo Syntax,” Plant Physiology, vol. 180, no. 1, pp. 39–55, May 2019, doi: https://doi.org/10.1104/pp.18.01401. Available: http://www.plantphysiol.org/content/180/1/39. [Accessed: Aug. 13, 2020]
[28] A. J. McCormick, P. Bombelli, R. W. Bradley, R. Thorne, T. Wenzel, and C. J. Howe, “Biophotovoltaics: oxygenic photosynthetic organisms in the world of bioelectrochemical systems,” Energy & Environmental Science, vol. 8, no. 4, pp. 1092–1109, 2015, doi: https://doi.org/10.1039/c4ee03875d
[29] Gaspar Taroncher-Oldenburg, Koren Nishina, and G. Stephanopoulos, “Identification and Analysis of the Polyhydroxyalkanoate-Specific β-Ketothiolase and Acetoacetyl Coenzyme A Reductase Genes in the Cyanobacterium Synechocystis sp. Strain PCC6803,” Applied and Environmental Microbiology, vol. 66, no. 10, pp. 4440–4448, Oct. 2000, doi: https://doi.org/10.1128/aem.66.10.4440-4448.2000




